pages/snek/detail/_id.vue

```diff
@@ -1,28 +0,0 @@
-<template>
-  <GalleryItem />
-</template>
-
-<script lang="ts">
-import { Component, Vue } from 'nuxt-property-decorator'
-
-import GalleryItem from '@/components/bsx/Gallery/Item/GalleryItem.vue'
-
-@Component<GalleryItemPage>({
-  name: 'GalleryItemPage',
-  components: {
-    GalleryItem,
-  },
-  head() {
-    return {
-      link: [
-        {
-          hid: 'canonical',
-          rel: 'canonical',
-          href: this.$root.$config.public.baseUrl + this.$route.path,
-        },
-      ],
```

Comment:
I wonder how could we handle the head config then?

Reply:
`useNuxt2Meta`? and after Nuxt3 migration [useHead](https://nuxt.com/docs/getting-started/seo-meta)